media-libs/libogg/libogg-1.3.6.recipe

```diff
@@ -17,29 +17,33 @@ pipelines. This stream orientation is the major design difference over other \
 file-based container formats.
 "
 HOMEPAGE="https://www.xiph.org"
-COPYRIGHT="1994-2021 Xiph.Org Foundation"
+COPYRIGHT="1994-2025 Xiph.Org Foundation"
 LICENSE="BSD (3-clause)"
-REVISION="2"
+REVISION="1"
 SOURCE_URI="http://downloads.xiph.org/releases/ogg/libogg-$portVersion.tar.gz"
-CHECKSUM_SHA256="0eb4b4b9420a0f51db142ba3f9c64b333f826532dc0f48c6410ae51f4799b664"
+CHECKSUM_SHA256="83e6704730683d004d20e21b8f7f55dcb3383cdf84c0daedf30bde175f774638"
 
 ARCHITECTURES="all"
-SECONDARY_ARCHITECTURES="x86_gcc2 x86"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion="0.8.6"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+portVersionCompat="$portVersion compat >= ${portVersion%%.*}"
 
 PROVIDES="
-	libogg$secondaryArchSuffix = $portVersion
-	lib:libogg$secondaryArchSuffix = 0.8.4 compat >= 0
+	libogg$secondaryArchSuffix = $portVersionCompat
```

Comment:
I don't think `$portVersionCompat` is needed here. Just `$portVersion` is enough. The same also for the devel package.